packages/block-library/src/list/block.json

```diff
@@ -13,15 +13,6 @@
 			"default": false,
 			"__experimentalRole": "content"
 		},
-		"values": {
```

Comment:
we can't remove this, otherwise createBlock breaks. (old templates with this attribute)